packages/shared/styles/global.ts

```diff
@@ -0,0 +1,9 @@
+import { injectGlobal } from "emotion";
+
+export const injectGlobalCss = () => {
+  return injectGlobal`
+    .ReactVirtualized__Grid {
```

Comment:
I don’t feel like this is a good idea from an a11y standpoint. But I am happy to be proven otherwise